content/03.methods.md

```diff
@@ -12,26 +12,35 @@ We character normalized the main text by mapping visually identical Unicode code
-These articles were the first 200 English language "Journal" articles returned by the Springer API for each month, resulting in 2400 articles per year for 2005 through 2020.
-To get the author information for the cited articles, we queried the Springer API using the scraped DOI.
+After news articles were scraped and processed, the text was processed using the coreNLP pipeline [@doi:10.3115/v1/P14-5010] (version 4.2.0).
+The main purpose for using coreNLP was to identify named entities related to countries and quoted speakers.
+The full set of annotaters were: tokenize, ssplit, pos, lemma,ner, parse, coref, quote.
+We used the "statistical" algorithm to perform coreference resolution.
+All results were output to json format for further downstream processing.
+
+
+#### _Springer_ API
+
+_Springer_ was chosen over other publishers for multiple reasons: 1) it is a large publisher, second only to Elsevier; 2) it covers multiple subjects, in contrast to PubMed; 3) it has a large daily query limit (5000/day); and 4) it provided more author affiliation information than found in Elsevier. 
+We generated a comparative background set for supplemental analysis with the _Springer_ API by obtaining author information for research articles cited in the news.
+We selected a random set of articles to generate the _Springer_ background set.
+These articles were the first 200 English language "Journal" articles returned by the _Springer_ API for each month, resulting in 2400 articles per year for 2005 through 2020.
+To get the author information for the cited articles, we queried the _Springer_ API using the scraped DOI.
 For both API query types, the author names, positions, and affiliations for each publication were stored and are available in "all_author_country.tsv" and "all_author_fullname.tsv".
  
 ### Name Normalization
  
 Names were cast to lowercase and processed using the R package humaniformat [@https://cran.r-project.org/web/packages/humaniformat/index.html].
 humaniformat identifies if names are reversed (Lastname, Firstname), as well as identifies middle names.
-Since many last or first authorships may be non-names, we additionally filtered out any identified names if they match any of the following terms: "consortium", "group", "initiative", "team", "collab*", "committee", "center", "program", "author", or "institute".
-Furthermore, since many articles only contain first and last name initials, we remove any names with a "." or "-" and are less than four characters.
+Since many last or first authorships may be non-names, we additionally filtered out any identified names if they partially or fully match any of the following terms: "consortium", "group", "initiative", "team", "collab", "committee", "center", "program", "author", or "institute".
+Furthermore, since many articles only contain first and last name initials, we remove any names with a  "." or "-" and are less than four characters.
```

Comment:
```suggestion
Furthermore, since many articles only contain first and last name initials, we remove any names with a  "." or "-" and those less than four characters.
```